good-enough-practices-for-scientific-computing.tex

```diff
@@ -640,8 +677,12 @@ \subsection*{Acknowledgments}
   share more than half of their data and code, they are probably best
   managed together, while if you are building tools that are used in
   several projects, the common code should probably be in a project of
-  its own.
-
+  its own. Projects do often require their own organizational model, but the
```

Comment:
Projects do often require their own way of organizing content, but the points below